package.json

```diff
@@ -23,6 +23,7 @@
     "@milkdown/react": "^7.1.1",
     "@milkdown/theme-nord": "^7.1.1",
     "@milkdown/transformer": "^7.1.1",
+    "@milkdown/utils": "^7.0.0-next.6",
```

Comment:
We need to use the latest 7.1.1 utils. Could you do `npm install @milkdown/utils` and push the changes?